File: src/main/java/itmo/is/project/repository/module/ModuleBlueprintRepository.java
package itmo.is.project.repository.module;

import itmo.is.project.model.module.ModuleBlueprint;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.repository.NoRepositoryBean;

@NoRepositoryBean
public interface ModuleBlueprintRepository<T extends ModuleBlueprint> extends JpaRepository<T, Integer> {
}

File: src/main/kotlin/itmo/isproject/repository/module/ModuleBlueprintRepository.kt
package itmo.isproject.repository.module

import itmo.isproject.model.module.ModuleBlueprint
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.repository.NoRepositoryBean

@NoRepositoryBean
interface ModuleBlueprintRepository<T : ModuleBlueprint> : JpaRepository<T, Int>
